cli/README.md

```diff
@@ -98,6 +98,6 @@ When installed, the module is executed as a regular binary by the DC/OS CLI, wit
 $ <exename> <modulename> [args]
 ```
 
-So for example, if someone called `dcos kafka broker list`, the `dcos-service-cli` CLI module would be run as `dcos-service-cli kafka broker list` by the DC/OS CLI.
+So for example, if someone called `dcos hello-world pod list`, the `dcos-service-cli` CLI module would be run as `dcos-service-cli hello-world pod list` by the DC/OS CLI.
 
-The inclusion of `modulename` (`kafka` in this example) as an argument allows reuse of a single CLI module build across multiple packages. In particular this is used by the default CLI build which is shared across most packages.
+The inclusion of `modulename` (`hello-workd` in this example) as an argument allows reuse of a single CLI module build across multiple packages. In particular this is used by the default CLI build which is shared across most packages.
```

Comment:
(nit) spelling on `hello-workd`

Reply:
Thanks. I've corrected this and also replaced `hello-world` with `helloworld` in this doc. It should be the package name.